client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+class ContactFormDialogFieldEditButton extends PureComponent {
+	static propTypes = {
+		expanded: PropTypes.bool.isRequired
+	};
+
+	constructor() {
+		super();
+		this.state = { showTooltip: false };
+	}
+
+	render() {
+		const { translate } = this.props;
+		const classes = classNames( 'editor-contact-form-modal-field__edit', {
+			'is-expanded': this.props.expanded
+		} );
+
+		return (
+			<div className="editor-contact-form-modal-field__edit-wrapper foldable-card__expand" >
+				<Button
+					className="editor-contact-form-modal-field__edit-wrapper-button"
+					ref="editField"
+					borderless
+					onMouseEnter={ () => this.setState( { showTooltip: true } ) }
+					onMouseLeave={ () => this.setState( { showTooltip: false } ) }>
```

Comment:
We can similarly write handlers once in the class rather than in render. The tricky part is that we don't have autobinding so we need to set that up in the constructor.
```
constructor() {
    super( ...arguments );
    this.state = { showTooltip: false };
    this.handleMouseEnter = this.handleMouseEnter.bind( this );
    this.handleMouseLeave = this.handleMouseLeave.bind( this );
}

handleMouseEnter() {
   this.setState( { showTooltip: true } );
}

handleMouseLeave() {
   this.setState( { showTooltip: false } );
}
```